lib/rubocop/cop/style/format_string_token.rb

```diff
@@ -75,11 +75,11 @@ def message(detected_style)
 
         # rubocop:disable Style/FormatStringToken
         def message_text(style)
-          case style
-          when :annotated then 'annotated tokens (like `%<foo>s`)'
-          when :template then 'template tokens (like `%{foo}`)'
-          when :unannotated then 'unannotated tokens (like `%s`)'
-          end
+          {
```

Comment:
Any particular reason you didn't move this to a constant?

Reply:
Those are just pieces of the message, not particularly useful for the whole class, so I decided it is cleaner to left it in a method instead of extracting to the top-level constant.
Can extract into a constant, if asked.

Reply:
Got it.

Reply:
I'll repeat that I feel it's bad style to keep on recreating the same constant hash over and over. Wast of CPU cycles and when reading you have to check the whole hash wondering "Hum, is there anything dynamic in here"?